### DISCUSSION TITLE
How to create Ort::value from image without cv::dnn::blobFromImage.

### DISCUSSION THREAD

**Discussion by seungtaek94** [microsoft/onnxruntime#13063] [Other Q&A]
Hi.

Is it possible to create 4-dims Ort:value from image without unsing cv::dnn:blobFromImage?

**Comment by pranavsharma** [comment]
I'm not familiar with cv::dnn:blobFromImage, but you can definitely create a 4-dims OrtValue.

**Reply by seungtaek94** [reply]
Thanks for reply.

Can you give some example to create 4-dims(N, C, H, W) OrtValue using 3-ch color image?
